### DISCUSSION TITLE
Hammerdb，During the test duration, the TPM becomes 0

### DISCUSSION THREAD

**Discussion by springeee** [TPC-Council/HammerDB#156] [Q&A] [answered]
Hi All:

 I use Hammerdb test Oracle Database.
the step as follows:
1, create table and user.
2, Schema Build
![image](https://user-images.githubusercontent.com/72604063/95651504-7502b180-0b1d-11eb-9e21-fcd0b5c4a58b.png)
3,Driver Script
![image](https://user-images.githubusercontent.com/72604063/95651522-94014380-0b1d-11eb-9af9-2dc7f6896686.png)
4, virtual user
![image](https://user-images.githubusercontent.com/72604063/95651534-a54a5000-0b1d-11eb-9229-ce987a7c8063.png)
5, run (Timing test period of 5 in minutes, about 2 minutes, the TPM become to 0)
![image](https://user-images.githubusercontent.com/72604063/95651545-b1361200-0b1d-11eb-942a-de8a5b92c6a9.png)

![image](https://user-images.githubusercontent.com/72604063/95651480-4dabe480-0b1d-11eb-97c4-2e3e35411f7b.png)

Could you please give me some advices , what shall i do to make the virtual user  always run during the test duration let the TPM is not 0 ?

Thanks,
Regards
Springee

**Comment by sm-shaw** [comment, marked as answer]
Apoliges missed this Issue. Yes by default the total transactions per user is set to "1000000". That means each virtual user will run 1M transactions before completing successfully. That is what has happened in this case. This setting is there so that you can control the workload by combining this with the Iterations value in the virtual user settings eg set Iterations to 5 and the virtual user will login, run 1M transactions, logout and then login again and you will see the iterations value increase in the GUI, so in total it will run 5M transactions. If you only wish it to run longer after 1 login then just increase the Total Transactions per User to more than 1M.

**Comment by springeee** [comment]
Hi sm-shaw：

Thanks very much .

I see it.

Thanks.
